### ISSUE TITLE
POLYGONS

### ISSUE THREAD

**Issue by csimoncosmostechnologiesinc** [Jannik-Schilling/generate_swmm_inp#59]
Typo in g_s_defaults.py and generate_swmm_import_inp_file.py of the word 'Polygons' that should be wrote in capitals as 'POLYGONS'. The typo prevents to import the polygons and rise an error in the subcatchments that uses a function defined in the polygons section.

**Comment by Jannik-Schilling** [comment]
@csimoncosmostechnologiesinc: I couldn´t repoduce the error so far. Can you please provide some more Information:
- which version of the plugin are you using?
- what did you do? What does the error look like? Is there an error message?

**Comment by Jannik-Schilling** [comment]
756881b68f5b8e44cd1745ce390a63ddd257a224

**Comment by Jannik-Schilling** [comment]
Hello Cesar, thank you for your comment. I just discovered some more issues, because for the subcatchment section SWMM doesn´t need polygons at all...

**Comment by csimoncosmostechnologiesinc** [comment]
You are right. However, having the subcatchment boundaries for the import function is essential because it is the reason to use the QGIS plugin vs. the SWMM interface.

Best,

Cesar A. Simon, EIT, PhD
OSHA, NASSCO
Civil Engineer

[A picture containing logo  Description automatically generated]

DBE / MBE / SBE
Pittsburgh  ***@***.***,-79.9969302,17z/data=!3m1!4b1!4m5!3m4!1s0x8834f3e4b94aeb0d:0x6630550090453c64!8m2!3d40.4500713!4d-79.9969302> | ***@***.***,-76.8928585,17z/data=!3m1!4b1!4m5!3m4!1s0x89c8c13947da1cb3:0x39861b8184d37058!8m2!3d40.2771621!4d-76.8906698> | ***@***.***,-81.6893236,17z/data=!3m1!4b1!4m5!3m4!1s0x8830fa809f82f103:0xb536c6b5709ad50d!8m2!3d41.4997673!4d-81.6871349> | ***@***.***,-81.5277502,17z/data=!3m1!4b1!4m5!3m4!1s0x8830d623232da979:0xb570486c52da429f!8m2!3d41.0724959!4d-81.5255615> | ***@***.***,-85.7080744,17z/data=!3m1!4b1!4m5!3m4!1s0x8869730b65ea26d5:0xf3f46c70d5de71e4!8m2!3d38.2741325!4d-85.7058857>
700 River Ave., Suite 100, Pittsburgh, PA 15212
O: 412.321.3951 ext. 402
F: 412.321.3954

E: ***@***.******@***.***>
W: www.cosmostechnologiesinc.com<http://www.cosmostechnologiesinc.com/>

**Comment by Jannik-Schilling** [comment]
[d5dba58](https://github.com/Jannik-Schilling/generate_swmm_inp/commit/d5dba58207b37cfd36ed980a1775eba705339891)
